x/wasm/module_test.go

```diff
@@ -154,7 +154,9 @@ func TestHandleInstantiate(t *testing.T) {
 	}
 	res, err := h(data.ctx, msg)
 	require.NoError(t, err)
-	require.Equal(t, res.Data, []byte("1"))
+	var pStoreResp MsgStoreCodeResponse
```

Comment:
Yup, a few helpers would be nice for `assertStoreCodeResponse([]byte, uint64)` or such

Reply:
added